pkg/puppetdb/orderBy.go

```diff
@@ -0,0 +1,17 @@
+package puppetdb
+
+import "fmt"
+
+func (o OrderBy) toOrderParam() map[string]string {
```

Comment:
For consistency, I think we should call this toParams (the same as the pagination struct).   At some point you could imagine they may all support an interface